library/src/main/java/com/getstream/sdk/chat/viewmodel/ChannelViewModel.java

```diff
@@ -274,12 +274,12 @@ private void configThread(Message threadParentMessage_) {
                 }
             });
         } else {
-            channel.getReplies(threadParentMessage_.getId(), String.valueOf(10), null, new GetRepliesCallback() {
+            channel.getReplies(threadParentMessage_.getId(), String.valueOf(30), null, new GetRepliesCallback() {
```

Comment:
please change `channel.getReplies` to accept numbers, passing `String.valueOf(30)` is super duper ugly